logic/common/tiled_utils.py

```diff
@@ -4,72 +4,70 @@
 import zlib
 import numpy
 
+
+
+#--------------------------------------------------#
+'''Base64 string <-> Array'''
+
 def DecodeIntoTiles2d(encoded_str, row_width):
-    '''Takes a TILED layer encoded string and converts it into a tiles2d
-    
-    :param encoded_str: a TILED layer's 'data'. This is an encoded string using zlib base64 compression
-    :param row_width: integer indicating how many cells each row should have in the resultant tiles2d
-    :returns tiles2d: A 2d array of tile IDs
-    
-    Example: 
-    >>> DecodeIntoTiles2d("eAENw4cNACAMAKA66/r/XiGhRES12R1O0+X2eH1+BeAATw==", 4)
-    >>> [[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]]
     '''
-    
-    # Decode the base64 string
+    Takes a TILED layer encoded string and converts it into a tiles2d
+      encoded_str - string containing a layer's 'data' in TILED, zlib-encoded and base64-compressed
+      row_width   - integer indicating number of cells each row should contain
+      tiles2d     - 2d array of Tile IDs, first row is at top
```

Comment:
I'm not a fan of this type of spacing - it creates unseemly gaps in the description IMO. Let's just use one space after the variable name